doc/lua_api.txt

```diff
@@ -4121,7 +4121,7 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ collision_removal: if true then particle is removed when it collides,
     --  ^ requires collisiondetection = true to have any effect
         attached = ObjectRef,
-    --  ^ attached: if defined, makes particle positions relative to this object.
+    --  ^ attached: if defined, Particle positions, velocities and accelerations are relative to this object's position and yaw.
```

Comment:
'Particle' doesn't need a capital letter.
Please split this long line at <=80 columns.